generator.py

```diff
@@ -127,7 +127,7 @@ def parse_args():
     parser.add_argument('path', nargs="*", default=[Path('.')],
                         type=valid_path,
                         help='Paths in which to look for Gir.toml files')
-    parser.add_argument('--gir-files', dest="gir_files_path", default=DEFAULT_GIR_FILES_DIRECTORY,
+    parser.add_argument('--gir-files-directory', dest="gir_files_path", default=DEFAULT_GIR_FILES_DIRECTORY,
```

Comment:
Do we want to persist using `dest="gir_files_path"` or switch to `gir_files_directory` from your change?

Reply:
I kept your wording. But both are fine for me.